model/pytorch/dffml_model_pytorch/pytorch_accuracy_scorer.py

```diff
@@ -0,0 +1,74 @@
+import os
+
+import torch
+
+from dffml.base import config
+from dffml.source.source import Sources
+from dffml.util.entrypoint import entrypoint
+from dffml.model import ModelNotTrained, ModelContext
+from dffml.accuracy import (
+    AccuracyScorer,
+    AccuracyContext,
+)
+
+
+@config
+class PytorchAccuracyConfig:
+    pass
+
+
+class PytorchAccuracyContext(AccuracyContext):
+    """
+    Accuracy Scorer for Pytorch Network Models
+    """
+
+    async def score(self, mctx: ModelContext, sctx: Sources):
```

Comment:
Pinging @sakshamarora1 for review of this

Reply:
The pytorchscore works :D